packages/gatsby-theme-docs/src/components/seo.js

```diff
@@ -9,9 +9,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Helmet } from 'react-helmet';
 import { useSiteData } from '../hooks/use-site-data';
+import useTopMenuItems from '../hooks/use-top-menu-items';
+
+const getSiteContextTitleByPath = (sitePathsMap, sitePath) => {
```

Comment:
I think the logic is the same as implemented in `layout-header.ts`. If that's the case, instead of duplicating the function, you could import it from the other file, or if you prefer, you could move it to a util module.